### PR TITLE
DRAFT decrease audio delay

### PR DESCRIPTION
**Pull request by seanshpark**
experiment to decrease audio delay.

### CODE REVIEW

**Comment by seanshpark** [comment]
@audetto , can you please check this if the audio delay is decreased?
what I changed is just reduced audio buffer size.

File: source/Speaker.cpp [hunk 1]
```diff
@@ -54,7 +54,7 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 //
 
 static const unsigned short g_nSPKR_NumChannels = 1;
-static const DWORD g_dwDSSpkrBufferSize = MAX_SAMPLES * sizeof(short) * g_nSPKR_NumChannels;
+static const DWORD g_dwDSSpkrBufferSize = MAX_SAMPLES * sizeof(short) * g_nSPKR_NumChannels * 2;
```
Comment (seanshpark) [inline source/Speaker.cpp:57]: increased speaker buffer to reduce degradation of sound

File: source/frontends/sdl/sdirectsound.cpp [hunk 2]
```diff
@@ -158,7 +159,8 @@ namespace
     const Uint8 svolume = Uint8(linVolume * SDL_MIX_MAXVOLUME);
 
     // this is a bit of a waste copy-time, but it reuses SDL to do it properly
-    myMixerBuffer.resize(size);
+    if (myMixerBuffer.size() < size)
+      myMixerBuffer.resize(size);
```
Comment (seanshpark) [inline source/frontends/sdl/sdirectsound.cpp:163]: not related with audio

File: source/frontends/sdl/sdirectsound.cpp [hunk 3]
```diff
@@ -89,7 +89,8 @@ namespace
     want.freq = myBuffer->sampleRate;
     want.format = AUDIO_S16LSB;
     want.channels = myBuffer->channels;
-    want.samples = 4096;  // what does this really mean?
+    // samples for 20 msec: 44100Hz * 2Ch * 0.02 => 1764
+    want.samples = Uint16(want.freq * want.channels * 0.02);
```
Comment (seanshpark) [inline source/frontends/sdl/sdirectsound.cpp:93]: this seems incorrect
```suggestion
    // samples for 40 msec: 44100Hz * 0.04 => 1764
    want.samples = Uint16(want.freq * 0.04);
```

**Comment by audetto** [comment]
I will probably merge this: https://github.com/audetto/AppleWin/pull/87

Could you please try it.

**Comment by audetto** [comment]
Superseded by https://github.com/audetto/AppleWin/pull/87
Please try now.

**Pull request closed without merging**